pinterestClone/ImagesListFolder/ImagesListViewController.swift

```diff
@@ -86,15 +105,22 @@ extension ImagesListViewController: UITableViewDelegate {
         let cellHeight = image.size.height * scale + imageInsets.top + imageInsets.bottom
         return cellHeight
     }
+    
+    ///В этом методе вызываем метод fetchPhotosNextPage из ImagesListService
```

Comment:
после док-коммента пробел оставляют:
```suggestion
    /// В этом методе вызываем метод fetchPhotosNextPage из ImagesListService
```